spec/support/asn1_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Asn1Helper
+  def asn1_dig(obj)
+    if obj.respond_to?(:first) && obj.first.is_a?(Enumerable)
+      asn1_dig obj.first
+    elsif obj.respond_to?(:value) && obj.value.is_a?(Enumerable)
+      asn1_dig obj.value
+    else
+      obj
+    end
```

Comment:
I wonder if we shouldn't just specifically do conditional assertion on `OpenSSL::VERSION`? What do you think?